src/NoFrixion.MoneyMoov/Extensions/PaymentRequestExtensions.cs

```diff
@@ -144,24 +144,29 @@ public static IEnumerable<PaymentRequestPaymentAttempt> GetCardPaymentAttempts(t
                             })
                             .ToList();
                 }
-                
-                if (cardCaptureEvent.Status == CardPaymentResponseStatus.CARD_AUTHORIZED_SUCCESS_STATUS || 
-                    cardCaptureEvent.Status == CardPaymentResponseStatus.CARD_PAYMENT_SOFT_DECLINE_STATUS)
+
+                if (cardCaptureEvent.EventType == PaymentRequestEventTypesEnum.card_sale &&

```

Comment:
This will now exclude ard authorize only payments. Is that intended?

Reply:
I believe it's because we only want to set the settled amount as the event amount for sales, as captures are calculated differently.

Reply:
The other if condition is handling AuthorizeOnly i.e a list of captures.